components/edit-collective/sections/HostVirtualCards.js

```diff
@@ -301,9 +326,11 @@ const HostVirtualCards = props => {
         </P>
         <Flex mt={3} flexDirection={['row', 'column']}>
           <VirtualCardFilters
+            isCollectiveFilter={true}
```

Comment:
I think you can remove this property if it is always true.

Reply:
@kewitz : I think we can't remove this, because we only want to show the collective filter on the virtual card page in the host settings. When it's the collective's virtual card we don't show this. So, for

https://github.com/opencollective/opencollective-frontend/blob/a18eebbb77f415100438f3fb07072d6ff76519d9/components/edit-collective/sections/virtual-cards/VirtualCards.js#L111

we don't have this property.